src/components/Delegation/dashboard/DelegatedStakepoolInfo.js

```diff
@@ -80,12 +127,25 @@ const DelegatedStakepoolInfo = ({
 
 export default injectIntl(
   (compose(
+    withStateHandlers(
+      {
+        showCopyNotif: false,
+      },
+      {
+        setshowCopyNotif: () => (showCopyNotif) => ({showCopyNotif}),
+      },
+    ),
     withHandlers({
       openExternalURL: ({poolURL}) => () => {
         if (poolURL) {
           Linking.openURL(poolURL)
         }
       },
+      copyPoolHash: ({poolHash, setshowCopyNotif}) => () => {
+        Clipboard.setString(poolHash)
+        setshowCopyNotif(true)
+        setTimeout(() => setshowCopyNotif(false), 10000)
```

Comment:
Why 10 seconds here?

Reply:
To avoid undesired rendering. Basically any additional tap within those 10 s won't generate an additional rendering of 'copied!' element. It'll be displayed 5 s and only after 5 more seconds a subsequent tab will render the message again.

Reply:
Is this the recommended way to do it over throttle/debounce?

Reply:
good catch Seba. this feels like a classic debounce case tho so maybe something like this? https://medium.com/@devmrin/debouncing-touch-events-in-react-native-prevent-navigating-twice-or-more-times-when-button-is-90687e4a8113

Reply:
Ah! I wasn't aware of those patterns (apart from intuition), nor that there are well-known implementations, thanks for the heads up! After reading a bit about it, I agree, it should be called with debounce and with the `leading` flag set. However, I still need that `setInterval` so that the component is actually re-rendered after a state change. Pushed changes.